.github/workflows/extensive_ci.yml

```diff
@@ -165,15 +165,19 @@ jobs:
 
     - name: Install simsopt package
       if: contains(matrix.packages, 'spec') || contains(matrix.packages, 'all')
-      run: pip install -v .[MPI,SPEC]
+      run: |
+        pip install -v .
+        pip install mpi4py py_spec pyoculus h5py
 
     - name: Install simsopt package
       if: contains(matrix.packages, 'none')
       run: pip install -v .
 
     - name: Install simsopt package
       if: contains(matrix.packages, 'vmec')
-      run: pip install -v .[MPI]
+      run: |
+        pip install -v .
+        pip install mpi4py
```

Comment:
What wasn't working about the original line here? Doesn't [MPI] work for installing mpi4py?

Reply:
For some reason, mpi4py was not getting installed. The runner is acting very peculiar and I am not sure why.